packages/widgets/src/components/NotificationCard/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IMapIcons, MainColors } from '@ssa-ui-kit/core';
+
+export interface NotificationCardProps {
+  title: React.ReactNode;
```

Comment:
It's not correct to use here `ReactNode` type, in my opinion.
Because it includes `null` and `undefined` types. We don't want those types for our `title` and `text` props.

Reply:
done